webrtc/RTCPeerConnection-helper.js

```diff
@@ -418,6 +418,28 @@ function getUserMediaTracksAndStreams(count, type = 'audio') {
   });
 }
 
+// A dependency agnostic way of adding track depending
+// on the supported method on a browser. Uses addTransceiver(track)
```

Comment:
Rewrite this to be more readable:
* If addTransceiver(track) is available, use that.
* If addtrack(track, mediastream) is available, use that.
* Otherwise, use addStream(mediastream).